### PR TITLE
fix(test): Fix date picker test and add new test

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

**Comment by mxiao6** [comment]
Add new test for #1572

File: src/components/date-picker/__tests__/DatePicker.test.js [hunk 1]
```diff
@@ -20,18 +38,6 @@ describe('components/date-picker/DatePicker', () => {
         clock.restore();
     });
 
-    test('should correctly render children in DatePicker', () => {
```
Comment (ConradJChan) [inline src/components/date-picker/__tests__/DatePicker.test.js]: Are there any corresponding snapshots to remove?

Comment (mxiao6) [reply src/components/date-picker/__tests__/DatePicker.test.js]: The corresponding snapshots have already been removed.

**Pull request merged**